external/storm-hive/src/main/java/org/apache/storm/hive/bolt/HiveBolt.java

```diff
@@ -134,22 +131,16 @@ public void execute(Tuple tuple) {
                     collector.ack(t);
                 tupleBatch.clear();
             }
+        } catch(SerializationError se) {
+            LOG.info("Serialization exception occurred, tuple is acknowledged but not written to Hive.", tuple);
+            collector.ack(tuple);
         } catch(Exception e) {
             this.collector.reportError(e);
             collector.fail(tuple);
-            try {
-                flushAndCloseWriters();
-                LOG.info("acknowledging tuples after writers flushed and closed");
-                for (Tuple t : tupleBatch)
-                    collector.ack(t);
-                tupleBatch.clear();
-            } catch (Exception e1) {
-                //If flushAndClose fails assume tuples are lost, do not ack
-                LOG.warn("Error while flushing and closing writers, tuples will NOT be acknowledged");
-                for (Tuple t : tupleBatch)
-                    collector.fail(t);
-                tupleBatch.clear();
-            }
+            for (Tuple t : tupleBatch)
```

Comment:
Could you wrap the body of this in '{}' I know it is just one line, but for coding conventions sake.